### ISSUE TITLE
Deploy with Coolify

### ISSUE THREAD

**Issue by angelocala94** [hcengineering/huly-selfhost#16]
Hi everyone,

Has anyone successfully deployed Huly Self-Hosted using [Coolify](https://coolify.io/)?

**Comment by grogou** [comment]
No there is no way for now I guess.

Dear developers, can we have a support for Coolify?

Thank you)

**Comment by christophberger** [comment]
@angelocala94 I got Huly up and running with Coolify:

- Create a new resource, select "docker compose file" as configuration source
- Copy template.compose.yaml and paste it into the docker compose file field
- Set two environment variables: `SERVER_ADDRESS=<server_ip>` and `HULY_VERSION=latest`
- Deploy
- Open `http://<server_ip>:8087`

**Comment by Jesuloba-world** [comment]
@angelocala94 were you able to access WS with an https URL generated by traefik

**Comment by christophberger** [comment]
No, but TBH, I didn't even try. I found some features missing in the self-hosted version (notably, GitHub integration and Virutal Office) and removed the Huly resource from my Coolify project. Looking for alternatives now.

**Comment by Ankan002** [comment]
I already have a Coolify instance ready. But I also want to deploy huly via Traefik and coolify is already using Traefik. How to make it possible?

**Comment by matifanger** [comment]
> [@Jesuloba-world](https://github.com/Jesuloba-world) No, but TBH, I didn't even try. I found some features missing in the self-hosted version (notably, GitHub integration and Virutal Office) and removed the Huly resource from my Coolify project. Looking for alternatives now.

did you found any alternative?

**Comment by christophberger** [comment]
@matifanger I stopped looking. My situation changed and I don't need a software of this kind anymore.
